R/src.presto.R

```diff
@@ -0,0 +1,62 @@
+# Copyright (c) 2015-present, Facebook, Inc.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree. An additional grant
+# of patent rights can be found in the PATENTS file in the same directory.
+
+#' dplyr integration to connect to a Presto database.
+#'
+#' Allows you to connect to an existing database through a presto connection.
+#'
+#' @param catalog Catalog to use in the connection
+#' @param schema Schema to use in the connection
+#' @param user User name to use in the connection
+#' @param host Host name to connect to the database
+#' @param port Port number to use with the host name
+#' @param parameters Additional parameters to pass to the connection
+#' @param ... Other arguments passed on to the underlying
+#'   database connector \code{dbConnect}
+#' @export
+#' @examples
+#' \dontrun{
+#' # To connect to a database
+#' my_db <- src_presto(catalog = "hive", schema = "web", user = "onur",
+#'   host = "localhost", port = 8888)
+#' }
+src_presto <- function(
+    catalog=NULL,
+    schema=NULL,
+    user=NULL,
+    host= NULL,
+    port=NULL,
+    parameters=NULL,
+    ...
+  ) {
```

Comment:
It might be good to check that dplyr is installed and give a descriptive error message otherwise here. It will still fail in line 49 otherwise, but with a more generic message.